pkg_resources/__init__.py

```diff
@@ -1141,8 +1141,7 @@ def obtain(self, requirement, installer=None):
         None is returned instead.  This method is a hook that allows subclasses
         to attempt other ways of obtaining a distribution before falling back
         to the `installer` argument."""
-        if installer is not None:
-            return installer(requirement)
+        if installer return installer(requirement) else None
```

Comment:
This is probably one of the things that is causing problems (syntax error).

Reply:
Fixed.